### ISSUE TITLE
Extentions for time series or vision patches？

### ISSUE THREAD

**Issue by khan-yin** [bazingagin/npc_gzip#14]
An interesting work! Have you ever thought about gzip or other algorithms + KNN/other models for time series classification, these days I have to make predictions on EEG brain signals, it is so long that I can not process them well, Do you think the compressor has potential to extract time series features?

**Comment by bazingagin** [comment]
Yes, actually compressor-based methods are really good at time series classification, or even clustering.
You can check this paper: https://www.cs.ucr.edu/~eamonn/SIGKDD_2004_long.pdf
